### PR TITLE
docs: Update Local_Development.mdx

### PR DESCRIPTION
**Pull request by Zamiell**
## PR Checklist

- [ ] Addresses an existing open issue: fixes #000
- [ ] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [X] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

Making docs more explicit. Future PRs should add better error messages to make reading the docs unnecessary.

### CODE REVIEW

File: docs/contributing/Local_Development.mdx [hunk 1]
```diff
@@ -86,7 +86,19 @@ You can run `yarn typecheck` in any package or in the root to run `tsc`.
...
+yarn install # To ensure that the dependencies are installed.
+yarn build # This builds all the packages except for `website-eslint` and `website`.
+cd packages/website-eslint
+yarn build
+cd ../..
+cd packages/website
+yarn build
+cd ../..
+```
+Next, running `yarn start` from either the root or `packages/website` will start a local dev server on `localhost:3000`.
```
Comment (kemilbeltre) [inline docs/contributing/Local_Development.mdx:101]: I think we just need to edit the notes below:

> The `website` package relies on other packages being built.
> We recommend running `yarn build` from the root, `packages/website-eslint`, and from this package  before `yarn start`.

**Review by kemilbeltre — COMMENTED** [review]
Nice update!

File: docs/contributing/Local_Development.mdx (hunk 1 above)
Comment (Zamiell) [reply docs/contributing/Local_Development.mdx:101]: yeah thanks

File: docs/contributing/Local_Development.mdx [hunk 2]
```diff
@@ -86,7 +86,16 @@ You can run `yarn typecheck` in any package or in the root to run `tsc`.
...
 Our interactive documentation website is built with [Docusaurus](https://docusaurus.io).
-Running `yarn start` from either the root or `packages/website` will start a local dev server on `localhost:3000`.
-
-> The `website` package relies on other packages being built.
-> We recommend running `yarn build` from the root before `yarn start`.
+After cloning the repository, run the following steps to install and build the website:
+
+```sh
+yarn install # To ensure that the dependencies are installed.
+yarn build # This builds all the packages except for `website-eslint` and `website`.
```
Comment (kemilbeltre) [inline docs/contributing/Local_Development.mdx:93]:
```suggestion
Before running `yarn start` you have to execute the following commands to build the necessary packages:

```sh
```

Comment (kemilbeltre) [reply docs/contributing/Local_Development.mdx:93]: In theory, you can't skip the setup.

Comment (Josh-Cena) [reply docs/contributing/Local_Development.mdx:93]: In this case, we should just say "remember to run the setup steps at the top of the page".

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Discord conversations are not a reliable source of change requests, even for docs 😄. Closing this because it's debatable whether the changes are useful. And the root of the issue is that `yarn build` doesn't build `website-eslint`.

**Pull request closed without merging**